src/tests/Interop/COM/ComWrappers/API/Program.cs

```diff
@@ -311,25 +315,26 @@ static void ValidateCreateObjectGcBehavior()
             IntPtr trackerObjRaw = MockReferenceTrackerRuntime.CreateTrackerObject();
 
             // Create the first native object wrapper and run the GC.
-            CreateObject();
-            GC.Collect();
```

Comment:
The use of `GC.Collect` instead of `ForceGC` was intentional. The bug in NativeAOT this test exposed was that a GC could null the contents of a weak GCHandle but not run finializers to remove that GCHandle from a cache. See #85235 . The `ForceGC` function runs finalizers, so this test would no longer be testing the same thing.

Reply:
> The use of GC.Collect instead of ForceGC was intentional.

Ah. That needs to be a comment then.

> The bug in NativeAOT this test exposed was that a GC could null the contents of a weak GCHandle but not run finializers to remove that GCHandle from a cache. 

Hmmm. This is a rather flakey test then. There is no guarantees the finalizer isn't going to be run. I will add notes to that effect. Thanks for the details.